{Doc} Add data collection statement (#16261)
Azure/azure-cli: README.md

For more usage examples, take a look at our [GitHub samples repo](http://github.com/Azure/azure-cli-samples) or [https://docs.microsoft.com/cli/azure/overview](https://docs.microsoft.com/cli/azure/overview).

> ## Data Collection

Reviewer: ```suggestion
## Telemetry
```

Author: The `Data Collection` title follows the [instruction](https://docs.opensource.microsoft.com/content/releasing/telemetry.html)

Reviewer: lesson learnt. Telemetry is used widely in vscode extensiosn repos,  learnt from there :)

The software may collect information about you and your use of the software and send it to Microsoft. Microsoft may use this information to provide services and improve our products and services. You may turn off the telemetry as described in the repository. There are also some features in the software that may enable you and Microsoft to collect data from users of your applications. If you use these features, you must comply with applicable law, including providing appropriate notices to users of your applications together with a copy of Microsoft's privacy statement. Our privacy statement is located at https://go.microsoft.com/fwlink/?LinkID=824704. You can learn more about data collection and use in the help documentation and our privacy statement. Your use of the software operates as your consent to these practices.

### Telemetry Configuration

> The collection of telemetry is currently enabled by default. If you want to turn it off, you can run `az config set core.collect_telemetry=no`. You can enable it back to help us improve our product by running `az config set core.collect_telemetry=yes`.

Reviewer: ```suggestion
Telemetry collection is on by default. To opt out, pls run `az config set core.collect_telemetry=no` to turn if off.
```
